bftengine/src/bcstatetransfer/RVBManager.cpp

```diff
@@ -710,6 +719,8 @@ void RVBManager::reportLastAgreedPrunableBlockId(uint64_t lastAgreedPrunableBloc
   DurationTracker<std::chrono::milliseconds> store_pruned_digests_dt("store_pruned_digests_dt", true);
   prune_report_in_progess_ = true;
   LOG_TRACE(logger_, KVLOG(lastAgreedPrunableBlockId, prune_report_in_progess_));
+  metrics_.pruning_callbacks_++;
```

Comment:
I suggest calling it prunning_reports
It's not really a callback.

Reply:
Thanks. Renamed.